src/cli.js

```diff
@@ -443,12 +444,16 @@ function cli (inputArgs) {
             });
         }
 
-        if (args.nosave) {
+        if (args.nosave || args['save-exact']) {
```

Comment:
this change is probably not needed since we handle the save or save-exact or no-save flags in cordova-fetch.

Reply:
kk updated! thanks!